include/xsimd/arch/xsimd_avx512bw.hpp

```diff
@@ -362,11 +362,11 @@ namespace xsimd
         {
             XSIMD_IF_CONSTEXPR(sizeof(T) == 1)
             {
-                return _mm512_mask_blend_epi8(cond, false_br, true_br);
+                return _mm512_mask_blend_epi8(cond, false_br.data, true_br.data);
```

Comment:
Error from gcc-11 from original code:
```bash
/home/cyb/xsimd/include/xsimd/types/../arch/./xsimd_avx512bw.hpp:365:24: error: cannot convert value to a vector
  365 |                 return _mm512_mask_blend_epi8(cond.data, false_br, true_br);
```

Reply:
how strange.

Reply:
Looks same issue as https://github.com/xtensor-stack/xsimd/issues/315.
I met with similar failures from SVE intrinsics, implicit cast breaks for no reason.